bounce.py

```diff
@@ -367,12 +368,15 @@ def _button_press_cb(self, win, event):
         win.grab_focus()
         x, y = list(map(int, event.get_coords()))
         self._press = self._sprites.find_sprite((x, y))
+        if self._press is not None and self._press == self.ball.ball:
```

Comment:
A bit strange looking, but what you are saying is if `self.ball.ball` is not None and `self._press` is the same.  Is there a better way to put this?

Reply:
Agreed, you can just compare `self._press` and `self.ball.ball` as it'll fail if either is `None`.